tailwind.css

```diff
@@ -78,6 +79,7 @@
     --blue-200: 216 228 245;
     --blue-100: 61 122 207;
     --blue-50: 61 122 207; /** replace with correct dark color ? **/
+    --blue-0: 61 122 207; /** replace with correct dark color ? **/
```

Comment:
@erikalogie dark color for 239 249 251? (light blue background color used for csv files)

Reply:
Yes that is the correct dark color